components/policy-editor/org.wso2.carbon.policyeditor/src/main/java/org/wso2/carbon/policyeditor/PolicyEditorService.java

```diff
@@ -173,6 +193,8 @@ public String formatXML(String xml) {
 
             xml = baos.toString("UTF-8");
 
+        } catch (ParserConfigurationException pce) {
+            throw new IllegalArgumentException("Failed to setup repository: ");
```

Comment:
exception object is not included in the wrapped exception. Provide some context information in the message.